src/analytics/private/sqls/redshift/dashboard/clickstream_user_first_attr_m_view_v1.sql

```diff
@@ -0,0 +1,16 @@
+CREATE MATERIALIZED VIEW {{schema}}.{{viewName}}
```

Comment:
This one is duplicated with `clickstream_user_first_attr_view_v1.sql`

Reply:
delete duplicated sql file